spk/wallabag/src/installer.sh

```diff
@@ -0,0 +1,66 @@
+#!/bin/sh
+
+# Package
+PACKAGE="wallabag"
+DNAME="Wallabag"
+
+# Others
+INSTALL_DIR="/usr/local/${PACKAGE}"
+WEB_DIR="/var/services/web"
+USER="$([ $(grep buildnumber /etc.defaults/VERSION | cut -d"\"" -f2) -ge 4418 ] && echo -n http || echo -n nobody)"
+TMP_DIR="${SYNOPKG_PKGDEST}/../../@tmp"
+
+
+preinst ()
+{
+    exit 0
+}
+
+postinst ()
+{
+    # Link
+    ln -s ${SYNOPKG_PKGDEST} ${INSTALL_DIR}
+
+    # Install the web interface
+    cp -R ${INSTALL_DIR}/share/${PACKAGE} ${WEB_DIR}
+
+    # permissions
+    chown -R ${USER} ${WEB_DIR}/${PACKAGE}
+
+    exit 0
+}
+
+preuninst ()
+{
+    exit 0
+}
+
+postuninst ()
+{
+    # Remove link
+    rm -f ${INSTALL_DIR}
+
+    # Remove the web interface
+    rm -fr ${WEB_DIR}/${PACKAGE}
+
+    exit 0
+}
+
+preupgrade ()
+{	
+	rm -fr ${WEB_DIR}/${PACKAGE}/cache/*
+	rm -fr ${WEB_DIR}/${PACKAGE}/install/
```

Comment:
L51 and L52 should not be necessary, as you presumably overwrite the contents via https://github.com/SynoCommunity/spksrc/pull/2165/files#diff-c0d645b343ac3265489c0734c80f5452R25.
Please verify and correct accordingly.